index.js

```diff
@@ -10,19 +10,27 @@ Geoservices.prototype.featureServer = function (req, res) {
   })
 }
 
+/**
+ * Handler for the $namepace/rest/info route. Inspects provider for authentation info and passes any on to the
+ * FeatureServer handler
+ * @param {*} req
+ * @param {*} res
+ */
 Geoservices.prototype.featureServerRestInfo = function (req, res) {
-  FeatureServer.route(req, res)
+  // Inspect provider for an "authInfo" controller; if undefined create a dummy function that returns an empty object
+  let authInfo = this.authInfo || function () { return {} }
```

Comment:
this needs to be `this.model.authInfo` or else it will always be undefined

Reply:
Moved functions off Controller and onto model in 7412260.